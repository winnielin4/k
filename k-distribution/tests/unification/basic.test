unify 3               and 3
unify 3               and 5
unify 3               and symbolicExp
unify 3 + 5           and 3 + symbolicExp
unify symbolicExp     and symbolicExp
unify symbolicExp + 5 and 3 + symbolicExp
