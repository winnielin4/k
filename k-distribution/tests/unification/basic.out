<generatedTop>
  <k>
    .
  </k>
  <results>
    ListItem ( unify ( 3 , 3 ) is SetItem ( .Map ) )
    ListItem ( unify ( 3 , 5 ) is .Set )
    ListItem ( unify ( 3 , V0 ) is SetItem ( V0 |-> 3 ) )
    ListItem ( unify ( 3 + 5 , 3 + V1 ) is SetItem ( V1 |-> 5 ) )
    ListItem ( unify ( V2 , V3 ) is SetItem ( V2 |-> V3 ) )
    ListItem ( unify ( V4 + 5 , 3 + V5 ) is SetItem ( V4 |-> 3
    V5 |-> 5 ) )
  </results>
</generatedTop>
